activiti-core/activiti-bpmn-converter/src/main/java/org/activiti/bpmn/converter/util/BpmnXMLUtil.java

```diff
@@ -154,9 +154,12 @@ public static ExtensionElement parseExtensionElement(XMLStreamReader xtr) throws
     while (!readyWithExtensionElement && xtr.hasNext()) {
       xtr.next();
       if (xtr.isCharacters() || XMLStreamReader.CDATA == xtr.getEventType()) {
-        if (StringUtils.isNotEmpty(xtr.getText().trim())) {
-          extensionElement.setElementText(xtr.getText().trim());
-        }
+          if (StringUtils.isNotEmpty(xtr.getText().trim())) {
+              if(StringUtils.isBlank(extensionElement.getElementText()))
```

Comment:
there's a space missing after the `if`, and it's a good practice to always put the curly braces `{}` to avoid unintentional errors

Reply:
Changed as suggested